src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -158,16 +158,44 @@
             <app-selectphase [phases]="filteredPhases"
               [phaseSelectionListType]="phaseSelectionListType"
               [phaseSelectionType]="phaseSelectionType"
-              [phaseSelected]="phaseSelected()"></app-selectphase>
-
-            <div class="form">
+              [phaseSelected]="phaseSelected()" ></app-selectphase>
+            <div id="Submission_option_hidden" class="row">
+              <div id="submission_option_style">
+            <b class="fw-bold content fs-14">Submission Options:</b>
+            <li>
+              <input
+                  type="radio" name="submissionOptions" class="with-gap selectPhase"
+                  id="fileUrl"  (click)="Oncheckboxclicked('fileUrl')">
+              <label for="fileUrl"></label>
+              <div id="radio_button_url" class="show-member-title pointer">
+                  <strong class="text-med-black">Enter file URL (for file size > 400MB) </strong>
+              </div>
+              
+              <input
+                  type="radio" name="submissionOptions" class="with-gap selectPhase"
+                  id="fileUpload" (click)="Oncheckboxclicked('fileUpload')">
+              <label for="fileUpload"></label>
+              <div id="radio_button_file" class="show-member-title pointer">
+                  <strong  class="text-med-black">Upload file </strong>
+              </div>
+            </li>
+              </div>
+            <div id="form_hidden" class="form">
               <div class="row row-lr-margin">
                 <div class="col-md-10 col-sm-10 col-lr-pad
                   submit-upload-file">
+                  <div id="uploadbutton" >
                   <app-input (change)="validateInput($event.target.value)"
                     [label]="'input_file'" [accept]="'.json, .zip, .txt, .tsv, .gz'"
                     [placeholder]="''" [type]="'file'" [value]="''"
                     #formsubmit></app-input>
+                  </div>
+                  <div id="url_field">
+                  <app-input (input)="validateInput($event.target.value)" 
+                    [label]="'submission_url'" [placeholder]="'Submission file URL'"
```

Comment:
Please update this label here due to which backend won't be able to get `file_url` value.